sources/academy/tutorials/api/run_actor_and_retrieve_data_via_api.md

```diff
@@ -84,6 +84,60 @@ If we press **Send**, it will immediately return some info about the run. The `s
 
 We will later use this **run info** JSON to retrieve the run's output data. This info about the run can also be retrieved with another call to the [**Get run**](https://apify.com/docs/api/v2#/reference/actors/run-object/get-run) endpoint.
 
+## JavaScript and Python client {#javascript-and-python-client}
+
+If you are using JavaScript or Python, it is highly recommended using Apify API client ([JavaScript](https://docs.apify.com/api/client/js/), [Python](https://docs.apify.com/api/client/python/)) instead of using the raw HTTP API. The client implements smart polling and exponential backoff which makes calling actors and getting results very simple.
```

Comment:
If you are using JavaScript or Python, we highly recommend using the Apify API client ([JavaScript](https://docs.apify.com/api/client/js/), [Python](https://docs.apify.com/api/client/python/)) instead of the raw HTTP API. The client implements smart polling and exponential backoff, which makes calling Actors and getting results very simple.